README.md

```diff
@@ -1 +1,5 @@
-# microverse-trials-review
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
We help our future collaborators and employers know what the project is about through the README.md file. Making it more detailed can make it more communicative and professional. I suggest you add things like; 

- Title of the project
- Author
- Tech stack
- Installation procedures
- Contribution guide etc